src/common/_modules/platform_service_bus/networking.tf

```diff
@@ -0,0 +1,12 @@
+resource "azurerm_subnet" "platform_service_bus_namespace_subnet" {
```

Comment:
Not needed, private connections are handled by the private endpoint created within the module

Reply:
removed in https://github.com/pagopa/io-infra/pull/1520/commits/e145a80ebc29b877f695c1e3fcc002c503c8f1fa